packages/zero-client/src/client/zero.ts

```diff
@@ -736,6 +743,14 @@ export class Zero<
     return createLogOptions(options);
   }
 
+  preload(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    query: Query<S, keyof S['tables'] & string, any>,
+    options: PreloadOptions,
+  ): void {
+    query.delegate(this.#zeroContext).preload(options);
```

Comment:
I think zero.preload makes some sense but do we want query.preload and query.run then?

We should either put things on zero or on query and it seems like we are moving towards having queries not be tied to a zero instance.

Reply:
I'm not sure yet. Still trying this out. I think we may want to keep `query.run / then` for use in custom mutators. Rather than:

```ts
await query.run(tx);
```

Reply:
The alternative could be:

```js
await tx.run(query);
```